ci/cases/yamls/gefs_replay_ci.yaml

```diff
@@ -0,0 +1,7 @@
+defaults:
+  !INC {{ HOMEgfs }}/parm/config/gefs/yaml/defaults.yaml
+base:
+  REPLAY_ICS: "YES"
+  FCST_BREAKPOINTS: ""
+  BASE_CPLIC: {{ 'TOPICDIR' | getenv }}/REPLAY_ICs/CI
```

Comment:
I think we'll need to have the replay IC data in the relevant directories on each platform. For example, on WCOSS2, I think we can have the replay IC data here: `/lfs/h2/emc/global/noscrub/emc.global/data/ICSDIR/`

Reply:
Yes. You can pass replay ICs to me and I will pull them into the `BASE_IC`/`../ICSDIR/` locations on each supported platform.

FYI, now that #2651 has been merged, you'll need to rename `BASE_CPLIC` to `BASE_IC`. See that PR for other changes that may need to be incorporated into this PR.

Reply:
Thank you, @KateFriedman-NOAA. The replay ICs can be found on Hera: /scratch2/NCEPDEV/stmp3/Neil.Barton/ICs/REPLAY_ICs/CI/2020110100

@NeilBarton-NOAA Please let us know if this is the incorrect location.

Reply:
@EricSinsky-NOAA I have copied the replay ICs here on Hera: `/scratch1/NCEPDEV/global/glopara/data/ICSDIR/C96C96mx100`

Let me know if the `C96C96mx100` folder name is correct, I got those values from the new `ci/cases/pr/C96_S2SWA_gefs_replay_ics.yaml` that this PR is adding. Also let me know if the ICs look good underneath and we'll sync them to the other platforms.

Lastly, I notice a couple files are size 0, is that intended?
```
[role.glopara@hfe12 C96C96mx100]$ ll gefs.20201101/00/mem000/* | grep " 0 Jun"
-rw-r--r-- 1 role.glopara global          0 Jun 12 17:03 20201101.030000.ca_data.nc
-rw-r--r-- 1 role.glopara global          0 Jun 12 17:03 20201101.030000.coupler.res
```

Reply:
@KateFriedman-NOAA Could you use just one C96? i.e., 

/scratch1/NCEPDEV/global/glopara/data/ICSDIR/C96mx100

Reply:
@NeilBarton-NOAA Yes, done: `/scratch1/NCEPDEV/global/glopara/data/ICSDIR/C96mx100`

BTW, I notice that all of the symlinks from mem001 to mem010 that point to mem000 have double `../../mem###/TYPE` in the paths (e.g. `../../mem000/ice/../../mem000/ice`):
```
[role.glopara@hfe12 ICSDIR]$ ll /scratch2/NCEPDEV/stmp3/Neil.Barton/ICs/REPLAY_ICs/CI/2020110100/mem001/ice/  
total 4
lrwxrwxrwx 1 Neil.Barton stmp 67 Jun 12 17:04 20201101.030000.cice_model.res.nc -> ../../mem000/ice/../../mem000/ice/20201101.030000.cice_model.res.nc
```

Reply:
@KateFriedman-NOAA the double soft links are due to the change in expected directory structure for the stage_ic task. I just soft linked the old structure to the new structure